.github/workflows/ci-test.yml

```
@@ -0,0 +1,42 @@
+name: CI-Test
```

actionlint

"jobs" section is missing in workflow

(syntax-check)